src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -431,13 +412,14 @@ protected void activate(final BundleContext context, final Config config) throws
 
     @Deactivate
     protected void deactivate() {
+
         if (this.locatorPathsTracker != null) {
             this.locatorPathsTracker.close();
             this.locatorPathsTracker = null;
         }
 
+        this.resourceBundleRegistry.close();
```

Comment:
Maybe it's good to add the null check. Not sure if "deactivate" is called after a failed "activate". That's the only scenario I see where `this.resourceBundleRegistry` could be `null`, however.

Reply:
I agree, implemented.